README.adoc

```diff
@@ -1,31 +1,39 @@
 :product: App Services
+:product-long-kafka: OpenShift Streams for Apache Kafka
+:product-long-registry: OpenShift Service Registry
 
+= {product} guides and samples
 
+== {product-long-kafka} guides
 
-= {product} Guides and Samples
+* link:./docs/kafka/getting-started-kafka[Getting started with {product-long-kafka}]
```

Comment:
General comment for all the links that include attributes: I think you might want to remove the `link` keyword, or the attributes won't resolve in the URLs? (The links should still work though.)

Reply:
Thanks, yes wasn't sure about this one. Didn't see that behavior in the existing broken URLs. Will clean up in fresh PR if needed.